src/mysql_k8s_helpers.py

```diff
@@ -189,6 +231,36 @@ def configure_instance(self) -> None:
             )
             raise MySQLConfigureInstanceError(e.message)
 
+    def reconfigure_instance(self) -> None:
+        """Reconfigure the instance to be used in an InnoDB cluster.
+
+        Assumes that the instance was previously configured, and that the cluster
+        admin user already exists.
+        """
+        options = {
+            "restart": "false",
+        }
+
+        configure_instance_command = f"dba.configure_instance('{self.server_config_user}:{self.server_config_password}@{self.instance_address}', {json.dumps(options)})"
```

Comment:
Why not use `configure_instance` function from `./lib/charms/mysql/v0/mysql.py`?
both of them run `dba.configure_instance`

Reply:
refactored `dba.configure_instance` to skip the creation of the clusteradmin user in 86b1309ee0fe242d44ce2a0d3360a0974b520bfd. this removes the need for the `reconfigure_instance` method